src/misc/cpan-faq.html

```diff
@@ -1038,147 +1042,178 @@ <h3>
     <li>
         <b>What version of Perl you are running?</b>
         <p>
-            Try <tt>perl -v</tt> at the command line to find out.
+            Type <code>perl -v</code> at the command line to find out.
         </p>
     </li>
     <li>
         <b>Are you running the latest released version of perl?</b>
         <p>
-            Look at <a href="http://www.perl.com/pub">http://www.perl.com/</a>
-            to find out. If it is not the latest released version, get that one
-            and see whether your bug has been fixed. Note that bug reports
-            about old versions of Perl, especially those prior to the 5.0
-            release, are less likely to be incorporated into the source as
-            Perl1 through Perl4 are largely unmaintained.
+            Look at <a href="https://www.perl.org/">https://www.perl.org/</a>
+            to find out. If you are not using the latest released version,
+            please try to replicate your bug on the latest stable release.
+        </p>
+        <p>
+            Note that reports about bugs in old versions of Perl, especially
+            those which indicate you haven't also tested the current stable
+            release of Perl, are likely to receive less attention from the
+            volunteers who build and maintain Perl than reports about bugs in
+            the current release.
+        </p>
+        <p>
+            This tool isn't appropriate for reporting bugs in any version prior
+            to Perl 5.0.
```

Comment:
Drop this paragraph, the odds of someone reporting a bug in perl4 now are pretty slim.

Reply:
Probably a good idea to drop it from the source docs as well.